### DISCUSSION TITLE
Is there a way to trigger on a pushed metric ?

### DISCUSSION THREAD

**Discussion by Gaibhne** [healthchecks/healthchecks#821] [Q&A]
Some of our jobs don't quite lend themselves to the 'call webhook after successful completion' metaphor for this or that reason. Many of the use cases I can think of could be trivially solved by having a hook/alert that instead of a boolean yes/no ping metaphor instead lets the ping push a metric, such as "Last 'success' log entry's date" or "Value of business thing that is supposed to change daily", and have alerts trigger on simple conditions based on that metric (think "pushed date older than 24 hours", "Downloaded and processed thingymagoos at the same count for more than 12 hours", etc).

Is that in any way possible, short of scripting it on my end and calling a hook only if I deem the conditions are satisfied ? I was thinking I could relatively trivially push things like last log file modified date, but having local logic evaluate the conditions is not very dead-man-switch thinking.

Maybe there already is a feature like that that I didn't find ?

**Comment by cuu508** [comment]
No, Healthchecks does not inspect ping payloads, it just stores them.

Check out https://sensorpad.io/ – I think it does precisely what you are describing – you can push datapoints (sensor readings), and you can set up simple or complex rules to evaluate sensor state on immediate values, on value aggregations, and so on.
